test-requirements.txt

```diff
@@ -1 +1,3 @@
 odoo_test_helper
+odoo-addon-base_global_discount @ git+https://github.com/OCA/server-backend.git@refs/pull/221/head#subdirectory=setup/base_global_discount
```

Comment:
having an error on :

Unreleased dependencies found in test-requirements.txt.

Reply:
Could you please remove this file in order to pass the tests.

Reply:
@GasparJarsa done